terraform-provider-constellation/examples/full/aws_cluster.tf

```diff
@@ -53,7 +53,7 @@ module "aws_infrastructure" {
       instance_type = "m6a.xlarge"
       disk_size     = 30
       disk_type     = "gp3"
-      initial_count = 2
+      initial_count = 3
```

Comment:
We should change this for all CSPs

Reply:
It is 3 not for all CSPs as far as I can see.

Reply:
It's 2 for GCP as far as I can see

Reply:
https://github.com/edgelesssys/constellation/blob/a4056bda5c840c1b0f2604c0ff191ae4559a2a12/terraform-provider-constellation/examples/full/gcp_cluster.tf#L59 ?